content/getting-started/tutorial/protobuf-client-tutorial.md

```diff
@@ -0,0 +1,214 @@
+option. That's fine: options don't change the contract. Here, the Protobuf
+compiler generates the C# code in namespace `MyProtobufClient` and contract-wise, it
+doesn't matter that the server uses a different namespace.
+
+### Program.cs - the client
+
+The main program starts by creating a connection to the server:
+
+```csharp
+// Create a simple console logger factory and configure the log level for category IceRpc.
+using ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
+    builder
+        .AddSimpleConsole()
+        .AddFilter("IceRpc", LogLevel.Debug));
+
+await using var connection = new ClientConnection(
+    new Uri("icerpc://localhost"),
+    clientAuthenticationOptions: null,
+    logger: loggerFactory.CreateLogger<ClientConnection>());
+```
+
+This connection naturally matches our server configuration:
+
+- the server address is `icerpc://localhost`, meaning we connect with the
+  `icerpc` protocol to `localhost` on the default port for `icerpc` (4062)
+- we don't specify a transport so we use the default multiplexed transport
+  (`tcp`)
+- the null `clientAuthenticationOptions` means we'll establish a plain
+  non-secure TCP connection
+
+{% callout %}
+
+Creating the connection instance does not establish the connection.
+
+{% /callout %}
+
+Then, the client program creates an invocation pipeline that flows into the
+connection:
+
+```csharp
+Pipeline pipeline = new Pipeline()
+    .UseLogger(loggerFactory)
+    .UseDeadline(defaultTimeout: TimeSpan.FromSeconds(10))
+    .Into(connection);
+```
+
+When we send a request, the request goes through the [Logger] interceptor and
+the [Deadline] interceptor before being sent over the connection. The `Deadline`
+interceptor we install here ensures the request times out after 10 seconds
+without a response.
+
+{% callout %}
+The Deadline interceptor communicates with the Deadline middleware via a request
+field sent alongside each request; this allows the middleware to enforce the
+deadline created by the interceptor. On the other hand, the Logger interceptor
+and middleware are totally independent.
+{% /callout %}
+
+Next, the client program creates a `Greeter` client with this invocation
+pipeline:
+
+```csharp
+var greeterClient = new GreeterClient(pipeline);
+```
+
+`GreeterClient` is a struct that the `protoc-gen-icerpc-csharp` generator generates
+from Protobuf service `Greeter`. This struct allows us to send requests to a remote
+service that implements `Greeter`.
+
+With this code, the address of the target service (or
+[service address][service-address]) is the default for `Greeter`, namely
+`icerpc:/visitor_center.Greeter`. It matches the route we created in the server.
+We could also create the same proxy with an explicit service address:
+
+```csharp
+var greeterClient = new GreeterClient(pipeline, new Uri("icerpc:/visitor_center.Greeter"));
+```
+
+Finally, the client sends a `Greet` request, awaits the response (the greeting),
```

Comment:
```suggestion
Finally, the client sends a `Greet` request, awaits the greeting response,
```

Reply:
"the greeting response" sounds like the response of the greeting, but here the response is the greeting.